test/scripts/openshift-ci/setup-e2e-tests.sh

```
@@ -95,48 +138,88 @@ if [[ "$1" =~ raw ]]; then
+  cat config/overlays/test/configmap/inferenceservice-openshift-ci-raw.yaml | \
+    sed "s/namespace: kserve/namespace: ${KSERVE_NAMESPACE}/" | \
+    envsubst | \
+    oc apply -f -
+  oc delete pod -n ${KSERVE_NAMESPACE} -l control-plane=kserve-controller-manager
+
+  # Patch DSC only in manual mode (operator mode uses yaml files directly)
+  if [[ "$INSTALL_ODH_OPERATOR" == "false" ]]; then
+    oc patch DataScienceCluster test-dsc --type='json' -p='[{"op": "replace", "path": "/spec/components/kserve/defaultDeploymentMode", "value": "RawDeployment"}]'
+  fi
 else
   export OPENSHIFT_INGRESS_DOMAIN=$(oc get ingresses.config cluster -o jsonpath='{.spec.domain}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)